core/modules/system/css/system.theme.css

```diff
@@ -94,6 +94,9 @@ tr.even .form-item {
   margin-bottom: 0;
   white-space: nowrap;
 }
+.ck-editor {
+  white-space: normal;
+}
```

Comment:
This would also work in core/modules/ckeditor5/css/ckeditor5.css

Worth considering?